packages/design-picker/src/components/theme-card/index.tsx

```diff
@@ -0,0 +1,129 @@
+import { Card, Popover } from '@automattic/components';
+import classnames from 'classnames';
+import { translate } from 'i18n-calypso';
+import { forwardRef, useMemo, useRef, useState } from 'react';
+import StyleVariationBadges from '../style-variation-badges';
+import type { StyleVariation } from '../../types';
+import type { Ref } from 'react';
+import './style.scss';
+
+interface ThemeCardProps {
+	name: string;
+	description?: string;
+	image: React.ReactNode;
+	imageClickUrl?: string;
+	imageActionLabel?: string;
+	banner?: React.ReactNode;
+	badge?: React.ReactNode;
+	styleVariations: StyleVariation[];
+	optionsMenu?: React.ReactNode;
+	isActive?: boolean;
+	isInstalling?: boolean;
+	isShowDescriptionOnImageHover?: boolean;
+	isSoftLaunched?: boolean;
+	onClick?: () => void;
+	onImageClick?: () => void;
+	onStyleVariationClick?: () => void;
+}
+
+const ThemeCard = forwardRef(
+	(
+		{
+			name,
+			description,
+			image,
+			imageClickUrl,
+			imageActionLabel,
+			banner,
+			badge,
+			styleVariations = [],
+			optionsMenu,
+			isActive,
+			isInstalling,
+			isShowDescriptionOnImageHover,
+			isSoftLaunched,
+			onClick,
+			onImageClick,
+			onStyleVariationClick,
+		}: ThemeCardProps,
+		forwardedRef: Ref< any > // eslint-disable-line @typescript-eslint/no-explicit-any
+	) => {
+		const e2eName = useMemo( () => name.toLowerCase().replace( /\s+/g, '-' ), [ name ] );
+		const imageRef = useRef< HTMLAnchorElement >( null );
+		const [ isShowTooltip, setIsShowTooltip ] = useState( false );
+
+		const isActionable = imageClickUrl || onImageClick;
+		const themeClasses = classnames( 'theme-card', {
+			'theme-card--is-active': isActive,
+			'theme-card--is-actionable': isActionable,
+		} );
+		const themeInfoClasses = classnames( 'theme-card__info', {
+			'theme-card__info--has-style-variations': styleVariations.length > 0,
+		} );
+
+		return (
+			<Card className={ themeClasses } onClick={ onClick } data-e2e-theme={ e2eName }>
```

Comment:
There's an opportunity to break these elements into separate components as ThemeCard gets more complex.